transport-classes-epoll/src/main/java/io/netty/channel/epoll/LinuxSocket.java

```diff
@@ -36,7 +37,8 @@
 /**
  * A socket which provides access Linux native methods.
  */
-final class LinuxSocket extends Socket {
+@UnstableApi
+public final class LinuxSocket extends Socket {
```

Comment:
how is this exposed as public ?

Reply:
The `EpollDomainSocketChannel` class is `public` and has a constructor that is `public` and take a `LinuxSocket` as an argument.